x11-misc/bumblebee/bumblebee-9999.ebuild

```diff
@@ -17,6 +18,8 @@ else
 	S="${WORKDIR}/Bumblebee-${COMMIT}"
 fi
 
+inherit ${SCM} autotools multilib readme.gentoo-r1 systemd user
+
 DESCRIPTION="Service providing elegant and stable means of managing Optimus graphics chipsets"
 HOMEPAGE="http://bumblebee-project.org https://github.com/Bumblebee-Project/Bumblebee"
 
```

Comment:
you should be able to nix the `--docdir` from the econf, as it's part of `EAPI 6`